plugins/dynamic-plugins-info/src/components/DynamicPluginsTable/DynamicPluginsTable.test.tsx

```diff
@@ -30,7 +30,7 @@ describe('DynamicPluginsTable', () => {
     // 6 mockapi returned external(enabled) + 53 internal(not enabled)
     // mockapi returns enabled plugins
     // keys from InternalPluginsMap are internal plugins
-    expect(await findByText('Plugins (59)')).toBeInTheDocument();
+    expect(await findByText('Plugins (61)')).toBeInTheDocument();
```

Comment:
why is this a gating test??

Reply:
In my understanding, this is a unit test combining some mock data and data from https://github.com/janus-idp/backstage-showcase/pull/1547/files#diff-106b7949dcc2210208b14a8b6bc9b821abfead95e1d565a3bb835ed96ce7f037
I guess that's fine, as `plugins/dynamic-plugins-info/src/components/InternalPluginsMap.tsx` was updated to include the 2 bulk import plugins as "internal".

Reply:
But I agree that could be improved, but not in the scope of this issue/PR.

Reply:
ok but there are 65 plugins in https://github.com/janus-idp/backstage-showcase/blob/9aeca2c1f2b53a5e33735d2b0d2a2e66cb920972/dynamic-plugins.default.yaml

So should this be updated to a valid value?

Reply:
@debsmita1 Thoughts?

Reply:
Since the CI isn't complaining, can we proceed with merging this and create a separate ticket to update the test case? I had to modify the test case in my PR 'cause the CI failed after I made changes to the `InternalPluginsMap.tsx` file.

Reply:
Please lgtm/approve it then if it's okay for you.

Reply:
FYI, I've created https://issues.redhat.com/browse/RHIDP-3793 to track this.